webapp/cypress/integration/loginActions.ts

```diff
@@ -11,7 +11,7 @@ describe('Login actions', () => {
         cy.log('**Redirects to error then login page**')
         cy.visit('/')
         cy.location('pathname').should('eq', '/error')
-        cy.get('button').contains('Log in').click()
+        cy.get('button').contains('Login').click()
```

Comment:
I don't understand why this changed. Everything in the `en.json` and the default value appear to be "Log in". 

https://github.com/mattermost/focalboard/blob/release-7.0/webapp/src/pages/loginPage.tsx#L68
```
                <Button
                    filled={true}
                    submit={true}
                >
                    <FormattedMessage
                        id='login.log-in-button'
                        defaultMessage='Log in'
                    />
                </Button>
```

Reply:
Since it's on the `/` route without being logged in, it's actually defined in the error: https://github.com/mattermost/focalboard/blob/f13d48568073739cccc83074ac7f36b5d27f63e1/webapp/src/errors.ts#L56

When I initially updated the i18n strings in [`a863e43` (#3071)](https://github.com/mattermost/focalboard/pull/3071/commits/a863e43fb0876d21d7996d92f5c3452a99c25437) the translation changed from `Log in` to `Login`

I believe this is due to `i18n-extract` not being enforced in CI _or_ that we're basing our branches off of 0.15 and the PR that modified `Log in` to `Login` in `errors.ts` targeted 0.16 (https://github.com/mattermost/focalboard/pull/2518)